packages/compile-solidity/src/compilerSupplier/loadingStrategies/VersionRange.ts

```diff
@@ -2,7 +2,7 @@ import debugModule from "debug";
 const debug = debugModule("compile:compilerSupplier");
 
 import requireFromString from "require-from-string";
-import originalRequire from "original-require";
```

Comment:
What was this `originalRequire` module doing previously?

Reply:
So `original-require` is a reference to Node's `require` as opposed to webpack's `require`. It was `require`ing the compiler and wrapping it using `solc/wrapper`. Reading the file from disk does the same thing, but differently.

Reply:
Is this change only for making it consistent with remote fetching of the compilers or there are more advantages of doing this?

Reply:
I think we are trying to get rid of usages of `original-require`. This would eliminate one more.